src/components/common/EntityInfo.vue

```diff
@@ -60,22 +60,22 @@ export default class EntityInfo extends Mixins(CommonMixin) {
   // Global getters
   @Getter(useStore) getBusinessId!: string
   @Getter(useStore) getBusinessInformation!: BusinessInformationIF
+  @Getter(useStore) getOriginalBusinessInfo!: BusinessInformationIF
   @Getter(useStore) getOriginalLegalName!: string
   @Getter(useStore) getBusinessContact!: ContactPointIF
-  @Getter(useStore) getEntitySnapshot!: EntitySnapshotIF
 
   /** The original entity type. */
   get originalEntityType (): string {
-    if (this.getEntitySnapshot?.businessInfo) {
-      return GetCorpFullDescription(this.getEntitySnapshot.businessInfo.legalType)
+    if (this.getOriginalBusinessInfo) {
+      return GetCorpFullDescription(this.getOriginalBusinessInfo.legalType)
     } else {
       return 'Unknown'
     }
   }
 
   /** The business number (aka tax id). */
   get businessNumber (): string {
-    return this.getBusinessInformation?.taxId
```

Comment:
`getBusinessInformation` always returns an object (it's initialized to empty properties in state-model.ts), so no need for conditional chaining.